src/main.cpp

```diff
@@ -5899,6 +5899,13 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
             pfrom->PushMessage(NetMsgType::SENDHEADERS);
         }
 
+        // Tell the peer what maximum xthin bloom filter size we will consider acceptable.
+        if (pfrom->ThinBlockCapable())
+        {
+            pfrom->PushMessage(
+                NetMsgType::FILTERSIZEXTHIN, (uint32_t)GetArg("xthinbloomfiltersize", MAX_BLOOM_FILTER_SIZE));
```

Comment:
Wonder if we could use a new global or some other way to pass the `xthinbloomfiltersize` 
(`p->nXthinBloomfilterSize` ?)

This will become handy in case we want to add an RPC command to change such value.

Reply:
I guess i'm of the opinion that changing the bloom filter size by RPC is a bit of overkill and not something that users will do except once every few years.  As for testing I would think we're just going to set it once to a high value so why go to the trouble.

Reply:
I'm surely biased cause I could imagine to have cases whene I qant to change the bloomfilter size to test how this will impact perfs, but as you said I don't see users using it much.